### ISSUE TITLE
Advanced HTML and CSS: Emphasize how the 3 week long course in ARs might be too big a detour.

### ISSUE THREAD

**Issue by VeruAmari** [TheOdinProject/curriculum#27258]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

In the Natural Responsiveness lesson at the Responsive Design section:

There is a 3 week long free course linked out in Additional Resources, which may be too big a deviation from the course.

Mentioning how it may be best to bookmark it for later and take it during Job Search instead could be a sensible approach.

### Path

Ruby / Rails, Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-natural-responsiveness#additional-resources

### (Optional) Discord Name

veruamari

### (Optional) Additional Comments

[ Link to the same issue in ruby path ](https://www.theodinproject.com/lessons/advanced-html-and-css-natural-responsiveness#additional-resources)

**Comment by CouchofTomato** [comment]
Hey @VeruAmari 

Optional resources are for those who want to dig a bit deeper so they may involve slightly longer courses. It's why we keep them out of the Assignment section. 

While I don't mind a clarfiying note. I'd prefer to avoid mentioning specifically learners should only do it during a job hunt. We can instead just note the course is long and should not block learners moving on to other lessons.

**Comment by VeruAmari** [comment]
I understand, adding a warning note on the course's length and how learners shouldn't let it hold back their progress in the curriculum sounds good!

**Comment by CouchofTomato** [comment]
Opening this up for help wanted.

**Comment by ParagPalkar** [comment]
As a learner myself, sometimes regardless of lengthy `Additional resources` I had to go through it and it turned out to be really very helpful to me. Which means for me it was correct decision to go through the Additional resources. 

In some cases it was not required to go through the Additional resources. But that I realized only when I already completed Additional resources halfway through. 

I believe it totally depends on the ability of the learner, which differs with individuals.

As a solution if we mention a NOTE clarifying that

> Take into notice the length of the course, we would suggest you to do it as per your convenience  

I believe to be able to do `Natural Responsiveness` it is necessary to do some practice, else it is quite difficult to understand. So i think mentioning something like I quoted might be helpful to the new learners.

If this approves, i would be happy to take the assignment.

**Comment by michellecaii** [comment]
Hi, Could I be assigned to work on this issue?

**Comment by CouchofTomato** [comment]
@ParagPalkar Apologies for the time spent in reviewing this. Happy in principal. Do you still want to work on it?

**Comment by Parag-develop** [comment]
@CouchofTomato Yes I want to work on it ! I request you to assign the task to me for this new account I am commenting from. I have lost login access to my previous @ParagPalkar due to 2FA credential lost.

**Comment by CouchofTomato** [comment]
@Parag-develop All yours
